shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -317,6 +317,20 @@ public final class MetricsNames {
     public static final String UPDATE_SUBSCRIBER_LATENCY = PREFIX + "controller.stream.update_subscriber_streamcut_latency_ms";  // Histogram
     public static final String UPDATE_SUBSCRIBER_FAILED = PREFIX + "controller.stream.update_subscriber_streamcut_failed";        // Counter and Per-stream Counter
 
+    // Controller EventProcessor framework event's latency Metrics
+    public static final String CONTROLLER_EVENT_PROCESSOR_DELETE_STREAM_LATENCY = PREFIX + "controller.eventProcessors.deleteStream.write_latency_ms";      // Histogram
+    public static final String CONTROLLER_EVENT_PROCESSOR_UPDATE_STREAM_LATENCY = PREFIX + "controller.eventProcessors.updateStream.write_latency_ms";      // Histogram
+    public static final String CONTROLLER_EVENT_PROCESSOR_SEAL_STREAM_LATENCY = PREFIX + "controller.eventProcessors.sealStream.write_latency_ms";      // Histogram
+    public static final String CONTROLLER_EVENT_PROCESSOR_TRUNCATE_STREAM_LATENCY = PREFIX + "controller.eventProcessors.truncateStream.write_latency_ms";      // Histogram
+    public static final String CONTROLLER_EVENT_PROCESSOR_AUTO_SCALE_STREAM_LATENCY = PREFIX + "controller.eventProcessors.autoScale.write_latency_ms";      // Histogram
+    public static final String CONTROLLER_EVENT_PROCESSOR_CREATE_READER_GROUP_STREAM_LATENCY = PREFIX + "controller.eventProcessors.createReaderGroup.write_latency_ms";      // Histogram
```

Comment:
I would recommend to use CONTROLLER_EVENT_PROCESSOR_CREATE_READER_GROUP_LATENCY instead of CONTROLLER_EVENT_PROCESSOR_CREATE_READER_GROUP_STREAM_LATENCY. Same for CONTROLLER_EVENT_PROCESSOR_DELETE_READER_GROUP_STREAM_LATENCY, CONTROLLER_EVENT_PROCESSOR_UPDATE_READER_GROUP_STREAM_LATENCY, CONTROLLER_EVENT_PROCESSOR_DELETE_SCOPE_STREAM_LATENCY.

Reply:
yes these names are more appropriate, fixed them